stglib/iq.py

```diff
@@ -91,142 +96,174 @@ def read_iq(filnam):
                     ds[k].attrs["units"] = iqmat["Data_Units"][k].replace("/s", " s-1")
             elif "_2_" in k or "_3_" in k:
                 ds[k] = xr.DataArray(
-                    iqmat[k][0:timelen, :], dims=("time", "cell_across")
+                    iqmat[k][0:timelen, :], dims=("time", "bin_across")
```

Comment:
Changed from cell_across to bin_across. Is this for consistency with other instruments, or something else?

Reply:
I changed it to be more consistent with the AQD and Sig. Also to be consistent with 'bindist' for utils.create_z.